ledger/participant-integration-api/src/main/scala/platform/apiserver/RateLimitingInterceptor.scala

```diff
@@ -56,19 +52,13 @@ private[apiserver] final class RateLimitingInterceptor(
       next: ServerCallHandler[ReqT, RespT],
   ): ServerCall.Listener[ReqT] = {
 
-    serviceOverloaded(call.getMethodDescriptor.getFullMethodName) match {
-      case Some(errorMessage) =>
-        val rpcStatus = com.google.rpc.Status
-          .newBuilder()
-          .setCode(Code.ABORTED.value())
-          .setMessage(errorMessage)
-          .build()
+    val fullMethodName = call.getMethodDescriptor.getFullMethodName
+    serviceOverloaded(fullMethodName) match {
 
-        logger.info(s"gRPC call rejected: $rpcStatus")
-
-        val exception = StatusProto.toStatusRuntimeException(rpcStatus)
-
-        call.close(exception.getStatus, exception.getTrailers)
+      case Some(damlError) =>
+        logger.info(s"gRPC call rate limited: $damlError [limited method is $fullMethodName]")
```

Comment:
By default, the instantiation of self-service error codes logs the error details/message. This will translate to double logging in this case. I would only leave the self-service error code logging. Wdyt?